packages/acm-dns-validated-cert/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin/
+/node_modules/
```

Comment:
Probably not needed, node modules shouldn't end up in this folder due to yarn workspaces

Reply:
True, but it can't hurt, and it's created by pulumi anyway.